vue-frontend/src/pages/portfolio/[id].vue

```diff
@@ -120,8 +123,8 @@ export default {
 };
 </script>
 
-<style lang="scss" scoped>
-@include media-breakpoint-up(lg) {
+<style lang="postcss" scoped>
+@screen lg {
   .parallax {
     height: 100vh;
```

Comment:
Shouldn't we need to apply tailwind classes here?

Reply:
Perspective property is not in tailwind css

Reply:
https://stackoverflow.com/a/49993773
https://tailwindcss.com/docs/translate
Refer these links

Reply:
but that perspective css property is not tailwind css